applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_element.cpp

```diff
@@ -1109,7 +1109,7 @@ void UPwSmallStrainElement<TDim,TNumNodes>::
     //Previous definitions
     const PropertiesType& rProp = this->GetProperties();
     const GeometryType& rGeom = this->GetGeometry();
```

Comment:
I do not know the whole code style in the GeoApp but this should be `r_geom` in the Structural for example :D

Reply:
See #10083 (Migrate the new designed elements from GeoMechanics branch to Kratos application).
I will move them to the Kratos core after that this issue is reviewed.

Reply:
rGeom is used as the default variable name in GroMechanics branch